packages/cli/src/commands/devHandler.js

```diff
@@ -14,6 +16,26 @@ import { generatePrismaClient } from '../lib/generatePrismaClient'
 
 const defaultApiDebugPort = 18911
 
+/**
+ * Finds a free port
+ * @param  {[Number]}   requestedPort Port to start searching from
+ * @param  {[Number[]]} excludePorts  Array of port numbers to exclude
+ * @return {[Number]}                 A free port equal or higher than requestedPort but not within excludePorts. If no port can be found then returns -1
```

Comment:
Just a heads up that I'm pretty sure you almost never wanna use the upper-cased types for the primitives. (See the box in this section here: https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#the-primitives-string-number-and-boolean)

```suggestion
 * @param  {[number]}   requestedPort Port to start searching from
 * @param  {[number[]]} excludePorts  Array of port numbers to exclude
 * @return {[number]}                 A free port equal or higher than requestedPort but not within excludePorts. If no port can be found then returns -1
```

Reply:
Thanks for teaching me this. I'll try to remember this for the rest of time now.